sycl/test-e2e/Regression/alloc_with_invalid_alignment.cpp

```diff
@@ -0,0 +1,77 @@
+// REQUIRES: level_zero
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+//==----- alloc_with_invalid_alignment.cpp - SYCL USM allocation test-----==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===--------------------------------------------------------------------===//
+
+#include <sycl/sycl.hpp>
+
+// Purpose of this test is to verify that when SYCL is backed by L0, the aligned
```

Comment:
We could simplify the wording. Something like:
"This test verifies that for L0 backend, aligned USM alloc functions returns null_ptr, when called with alignment values that are not a power-of-2. "

Reply:
Also, why just for L0 backend? What happens for opencl backend? If this test is supposed to run on a specific backend, it should have a "REQUIRES: opencl" or "REQUIRES:level_zero" statement

Reply:
There is a requires level_zero line in the first line of this file.

Reply:
Aahh.. I missed that. 👍🏻